tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -13,6 +13,9 @@ import scala.util.Random
 class OneAndTests extends CatsSuite {
   checkAll("OneAnd[List, Int]", OrderLaws[OneAnd[List, Int]].eqv)
 
+  checkAll("OneAnd[List, Int] with Option", TraverseTests[OneAnd[List, ?]].traverse[Int, Int, Int, Int, Option, Option])
+  checkAll("Functor[OneAnd[List, A]]", SerializableTests.serializable(Traverse[OneAnd[List, ?]]))
```

Comment:
Super minor, but Functor should probably be replaced with Traverse for the name of the Serializable test, to reflect the check being performed.